docs/contents/components/forms/select/index.mdx

```diff
@@ -73,6 +73,32 @@ To set a default value, set a string to `defaultValue`.
 </Select>
 ```
 
+### Header
```

Comment:
Change "Header" to "Add header to a dropdown".

Reply:
[f411e3d](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/f411e3d2a53f9a4c4c189a520e4215a58511cb6e)